src/App.js

```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
```

Comment:
- [ ] It is important to import ``React` from 'react' in this code.

- JSX, which is used in this code to define the structure of the component, requires React to be in scope. Without importing React, the code would throw possible ReferenceError when attempting to use JSX.

Reply:
Does that mean for each React component that I create I should always import React from' react'?

Reply:
@dazehinn, You no longer need to `import React from "react"`. Starting from the release of React 17 of React, JSX is automatically transformed without using `React.createElement`.

Reply:
@Gambit142, this is very enlightening. Thanks. I'm actuallly using React 18.